WalletWasabi.Gui/ViewModels/WasabiDocumentTabViewModel.cs

```diff
@@ -125,7 +113,6 @@ public void Select()
 		public async Task<object> ShowDialogAsync()
 		{
 			DialogResult = null;
```

Comment:
ShowDialog will be working after this?

Reply:
Wow! what a huge hack that is! I cant even make sense of that that is supposed to do?

Did we hack try to disable tab switching again? Sounds like that should be a completely different UI concept!

Reply:
Tested, it works.